Updated troubleshooting section (#16)
ReVanced-Extended-Community/Community-Guides: general-guides/community-wiki/yt-troubleshooting.md

# **YT ReVanced Extended Troubleshooting**

### **1. No internet connection / The home page isn't loading**

> Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.

Reviewer: On my device (Samsung, running A14, OneUI 6) my GmsCore account appears similar to my Google account, and it does not say "GmsCore"

Only the logo is different.

![Screenshot_20240521_141952_Settings](https://github.com/ReVanced-Extended-Community/Community-Guides/assets/84587632/932a0514-3efa-4d6c-a7e1-e01333b54e4e)

I don't see a simple way of explaining this to users without showing a screenshot of what the logo is logo is. LMK if I should create and upload an image (I've been doing it through imgur for some reason 🤷)

Author: Done

replaced it with Google (blue icon), is this good enough?

> > If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) installed.

Reviewer: The purpose of uninstalling is to forcefully remove the user's account from the device. So it should be:
 
```suggestion
> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).

```

### **2. I'm having issues signing in to my Google account**

Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.

> > If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) i

Reviewer: I think we can combine this issue with `#1`. So it can just say something like:

 ```
### **2. I'm having issues signing in to my Google account**

Follow the steps in issue **#1** above.
```

What do you think? (Same thing for the ytm troubleshooting)

Author: Done, check if it's good enough

### **3. The download feature isn't working**

> Refer to step 4 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2b-downloader-setup) to understand and set up the download function in YT ReVanced Extended.

Reviewer: When linking to a part of a good it is good practice to link to a general section (ie: section 2, instead of 2b) because the link may break if the page is changed.

```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YT ReVanced Extended.

```
